Title isn't shown instead "Oops, an error occurred"

This one isn't shown as single hit:
https://alma.lobid.org/resources/990054518430206441
Data ist there (append `.json):` https://alma.lobid.org/resources/990054518430206441.json

@blackwinter  points out that this resource  has 2k items, and that seems to be in fact the problem:

```
! @81boda180 - Internal server error, for (GET) [/resources/990054518430206441] ->

play.api.UnexpectedException: Unexpected exception[ArrayIndexOutOfBoundsException: 1]
        at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:261) ~[com.typesafe.play.play_2.11-2.4.11.jar:2.4.11]
     [...]
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
        at controllers.resources.Lobid.compareIsil(Lobid.java:409) ~[lobid-resources-web.lobid-resources-web-0.2.2-SNAPSHOT-sans-externalized.jar:na]
        at views.html.tags.items_map_Scope0$items_map$$anonfun$markers$1$1.apply(items_map.template.scala:46) ~[lobid-resources-web.lobid-resources-web-0.2.2-SNAPSHOT-sans-externalized.jar:na]
        at views.html.tags.items_map_Scope0$items_map$$anonfun$markers$1$1.apply(items_map.template.scala:46) ~[lobid-resources-web.lobid-resources-web-0.2.2-SNAPSHOT-sans-externalized.jar:na]
        at scala.math.Ordering$$anon$9.compare(Ordering.scala:200) ~[org.scala-lang.scala-library-2.11.11.jar:na]
        at java.util.TimSort.binarySort(TimSort.java:296) ~[na:1.8.0_151]
        at java.util.TimSort.sort(TimSort.java:239) ~[na:1.8.0_151]
        at java.util.Arrays.sort(Arrays.java:1438) ~[na:1.8.0_151]
        at scala.collection.SeqLike$class.sorted(SeqLike.scala:648) ~[org.scala-lang.scala-library-2.11.11.jar:na]
        at scala.collection.AbstractSeq.sorted(Seq.scala:41) ~[org.scala-lang.scala-library-2.11.11.jar:na]
        at scala.collection.SeqLike$class.sortWith(SeqLike.scala:601) ~[org.scala-lang.scala-library-2.11.11.jar:na]
        at scala.collection.AbstractSeq.sortWith(Seq.scala:41) ~[org.scala-lang.scala-library-2.11.11.jar:na]
        at views.html.tags.items_map_Scope0$items_map.markers$1(items_map.template.scala:46) ~[lobid-resources-web.lobid-resources-web-0.2.2-SNAPSHOT-sans-externalized.jar:na]
        at views.html.tags.items_map_Scope0$items_map.apply(items_map.template.scala:157) ~[lobid-resources-web.lobid-resources-web-0.2.2-SNAPSHOT-sans-externalized.jar:na]

```

--- a/web/app/controllers/resources/Lobid.java
+++ b/web/app/controllers/resources/Lobid.java
@@ -406,12 +406,16 @@ public class Lobid {
 				// use secondary if main is equal, e.g. DE-5-11 before DE-5-20
 				return numerical(all1[2]) < numerical(all2[2]);
 			}
-		} else if (all1[1].equals(all2[1])) {
-			// same main sigel, prefer shorter, e.g. DE-5 before DE-5-11
-			return all1.length < all2.length;
+		} else if (all1.length == 2 && all2.length == 2) {
+			if (all1[1].equals(all2[1])) {
+				// same main sigel, prefer shorter, e.g. DE-5 before DE-5-11
+				return all1.length < all2.length;
+			}
+			// compare by main sigel, e.g. DE-5 before DE-6:
+			return numerical(all1[1]) < numerical(all2[1]);
 		}
-		// compare by main sigel, e.g. DE-5 before DE-6:
-		return numerical(all1[1]) < numerical(all2[1]);
+		// not actually an ISIL, compare given values as numbers:
+		return numerical(i1) < numerical(i2);
 	}
 
 	private static int numerical(String s) {

